src/Infrastructure/Repository/TaskRepository.php

```diff
@@ -108,6 +109,43 @@ public function findAllIndexedCollection(): array
         return $tasksMap;
     }
 
+    /**
+     * @param \SprykerSdk\Sdk\Infrastructure\Entity\Task $task
+     *
+     * @return \SprykerSdk\SdkContracts\Entity\TaskInterface
+     */
+    protected function changePhpCommand(TaskInterface $task): TaskInterface
+    {
+        $existingCommands = [];
+
+        foreach ($this->existingTasks as $existingTask) {
+            $existingTaskCommands = $existingTask->getCommands();
+
+            if ($existingTask instanceof TaskSetInterface) {
+                $existingSubTaskCommands = array_map(fn (TaskInterface $subtask) => $subtask->getCommands(), $existingTask->getSubTasks());
+                $existingTaskCommands = array_merge($existingTaskCommands, ...$existingSubTaskCommands);
```

Comment:
Array merge inside the loops should be avoided

Reply:
I replaced with two loops.